src/test/java/org/apache/maven/plugins/jlink/JLinkMojoTest.java

```diff
@@ -61,4 +63,19 @@ void single_quotes_shell_command() throws Exception {
                 .isEqualTo(
                         "/bin/sh -c '/path/to/jlink \"--strip-debug\" \"--module-path\" \"foo:bar\" \"--add-modules\" \"mvn,jlink\"'");
     }
+
+    @Test
+    void single_quotes_shell_command_windows() throws Exception {
+        Assumptions.assumeTrue(System.getProperty("os.name").startsWith("Windows"));
```

Comment:
`@EnabledOnOs`

Reply:
Neat. Haven't seen that one  before. Done.